packages/data/tests/unit/adapters/facts/bard-test.ts

```diff
@@ -313,6 +313,60 @@ module('Unit | Adapter | facts/bard', function (hooks) {
       /Exactly one 'tableName.dateTime' filter is required, you have 2/,
       '_buildDateTimeParam throws an error for multiple datetime filters'
     );
+
+    const since: RequestV2 = {
+      ...EmptyRequest,
+      table: 'tableName',
+      filters: [
+        {
+          field: 'tableName.dateTime',
+          type: 'timeDimension',
+          parameters: {
+            grain: 'isoWeek',
+          },
+          operator: 'gte',
+          values: ['start'],
```

Comment:
Can we use a real date to demonstrate how this is/should-be used?

To make it clean we could do
```js
values[startDate].
....
 assert.deepEqual(
      Adapter._buildDateTimeParam(since),
      `${startDate}/9999-12-27T00:00:00.000`,
      '_buildDateTimeParam uses far future aligned to grain for since operator'
    );

```